src/main/resources/assets/simplycaterpillar/lang/en_US.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
This file name should be all lowercase.

Reply:
Fixed in commit https://github.com/The-Fireplace-Minecraft-Mods/Simply-Caterpillar/pull/4/commits/6a3dfb1b52f7d003d0d2b9d4846783319089f11d